gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDatasetFinder.java

```diff
@@ -40,18 +39,20 @@
  * and creates a {@link IcebergDataset} for each one.
  */
 @Slf4j
-@AllArgsConstructor
 public class IcebergDatasetFinder implements IterableDatasetFinder<IcebergDataset> {
 
   public static final String ICEBERG_DATASET_PREFIX = DatasetConstants.PLATFORM_ICEBERG + ".dataset";
   public static final String ICEBERG_HIVE_CATALOG_METASTORE_URI_KEY = ICEBERG_DATASET_PREFIX + ".hive.metastore.uri";
   public static final String ICEBERG_DB_NAME = ICEBERG_DATASET_PREFIX + ".database.name";
   public static final String ICEBERG_TABLE_NAME = ICEBERG_DATASET_PREFIX + ".table.name";
 
-  private String dbName;
-  private String tblName;
   private final Properties properties;
-  protected final FileSystem fs;
+  protected final FileSystem sourceFs;
+
+  public IcebergDatasetFinder(FileSystem fs, Properties properties) {
```

Comment:
note: equivalent to `@lombok.RequiredArgsConstructor`

Reply:
updated it to the lombok annotation :)!